docs/src/pages/components/box/box.md

```diff
@@ -49,7 +49,7 @@ The Box children accepts a render props function. You can pull out the `classNam
 
 ## The sx prop
 
-Sometimes, the props on the Box component are not enough to style the component. To solve this, `Box` supports the `sx` prop. This allows you to specify any CSS rules you want, in addition to the ones already available using system props. Here is an example of how you can use it:
+All system props, are available via the `sx` prop. This property allows you to specify any CSS rules you may need, in addition to the ones already available as part of the system. Here is an example of how you can use it:
```

Comment:
This section might need to be moved up and reworded, as now the examples use `sx` before it's introduced, while the introduction still talks about system.

Reply:
Agree, will restructure it a bit tomorrow

Reply:
Right, I've added this as a first example 👍